src/ssort/_graphs.py

```diff
@@ -122,7 +130,7 @@ def replace_cycles(graph, *, key):
             prev = node
 
 
-def is_topologically_sorted(nodes, graph):
+def is_topologically_sorted(nodes: list[_T], graph: Graph[_T]) -> bool:
```

Comment:
Even with `from __future__ import annotations`, I think it would be best to stick with constructs that work in all supported versions of python, i.e. `typing.List`.

Reply:
I have no idea how to verify that we are doing this, however.

Reply:
Hmm, my understanding is that using `list` is preferred over `typing.List`. `typing.List` was depreciated in 3.9 and `list` is supported by 3.7+ as long as `from __future__ import annotations` is included. I think it would be best to avoid depending on depreciated features if possible.

Reply:
You are right.  Time really flies.  Ok, `list[_T]` is good.